ang/civicase/case/contact-case-tab/directives/contact-case-tab-case-details.directive.html

```diff
@@ -113,7 +113,7 @@
         <span>
           <span
             class="civicase__contact-card civicase__contact-card--manager"
-            civicase-contact-card contacts="[item.manager]">
+            contact-card contacts="[item.manager]">
```

Comment:
are we removing the prefix from all directives?

Reply:
Sorry this was removed by accident, fixed.